task1/lib/main.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Center(
+          child: Text("Flutter Vadodara"),
+        ),
+      ),
+      body: Column(
+        children: [
+          Container(
+            color: Colors.blue,
+            padding: EdgeInsets.fromLTRB(80, 10, 10, 10),
+            margin: EdgeInsets.all(30),
+            width: 300.0,
+            height: 100,
+            child: Text("Flirt",
+                style: TextStyle(fontStyle: FontStyle.italic, fontSize: 60.0)),
+          ),
+          Container(
+            color: Colors.orange,
+            padding: EdgeInsets.fromLTRB(80, 10, 10, 10),
+            margin: EdgeInsets.all(30),
+            width: 300.0,
+            height: 100,
+            child: Text("With",
+                style: TextStyle(fontStyle: FontStyle.italic, fontSize: 60.0)),
+          ),
+          Container(
+            color: Colors.teal,
+            padding: EdgeInsets.fromLTRB(60, 10, 10, 10),
+            margin: EdgeInsets.all(30),
+            width: 300.0,
+            height: 100,
+            child: Text("Flutter",
+                style: TextStyle(fontStyle: FontStyle.italic, fontSize: 60.0)),
+          ),
+          Container(
```

Comment:
I don't need the `Raisedbutton`. Also don't wrap your `RaisedButton` with `Container`. It doesn't makes any sense.